src/Lumper.Lib/BSP/Lumps/GameLumps/StaticPropLump.cs

```diff
@@ -244,9 +222,23 @@ StaticPropVersion.V7 or
         // since v10
         if (ActualVersion >= StaticPropVersion.V10)
             writer.Write(prop.FlagsEx);
-        // since v11
-        if (ActualVersion >= StaticPropVersion.V11)
-            writer.Write(prop.UniformScale);
+
+        // v11/v12
+        if (ActualVersion is StaticPropVersion.V11 or StaticPropVersion.V12)
+        {
+            if (prop.UniformScale.X == prop.UniformScale.Y)
+                writer.Write(prop.UniformScale.X);
+            else
+                throw new InvalidDataException("UniformScale only supports one value in version 11/12");
+        }
+
+        if (ActualVersion >= StaticPropVersion.V13)
+        {
+            writer.Write(prop.UniformScale.X);
+            writer.Write(prop.UniformScale.Y);
```

Comment:
more v13

Reply:
Okay, split this up but messy, see above comment